lib/protoparser/vmimport/parser_test.go

```diff
@@ -229,3 +241,42 @@ garbage here
 		},
 	})
 }
+
+func TestRowsUnmarshalNaNSuccess(t *testing.T) {
```

Comment:
Would modifying `TestRowsUnmarshalSuccess` to support NaN comparison make it easier?